src/keystore/prf.rs

```diff
@@ -2,25 +2,35 @@
 
 use super::Error;
 use crypto::hmac::Hmac;
-use crypto::sha2::Sha256;
+use crypto::sha2::{Sha256, Sha512};
 use std::fmt;
 use std::str::FromStr;
 
 /// `HMAC_SHA256` pseudo-random function name
 pub const HMAC_SHA256_PRF_NAME: &'static str = "hmac-sha256";
```

Comment:
hmac seems to be a theme these days ;)... just a dumb question, but would there be any benefit (if it's even possible?) to using @sorpaas's new https://github.com/ethereumproject/hmac-drbg-rs here?

Reply:
For no-std maybe
But for crypto, I preffer proved/peer-reviewed libs.